taxi/models.py

```diff
@@ -0,0 +1,38 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    class Meta:
+        ordering = ["name"]
+
+    def __str__(self):
```

Comment:
You got so caught up in the bigger issues that you forgot to add type annotations to all the `__str__()` methods.  
 :)